docs/templates/modules/hub.md

```diff
@@ -30,27 +35,52 @@ On this page:
-    - **config** – Used for configuration settings. Include the following files within the template:
-      - [metadata.json](#metadatajson)
-    - **ms-cm-exports** – Used for Cost Management exports that have not been transformed.
 - Data factory – Used for data ingestion and processing. Depends on: Storage account.
   - **dataFactoryName** = `param:hubName + "-engine"`
-  - pipelines – Create the following nested pipelines:
-    - **ms-cm-exports_RemoveOldFiles** – Removes all old files in the same directory when a new file is added to the ms-cm-exports container.
-
-### metadata.json
-
-The **metadata.json** file is used to identify the toolkit version.
 
-```json
-{
-  "$schema": "https://aka.ms/finops/toolkit/metadata-schema",
-  "type": "HubInstance",
-  "version": "0.1",
-  "learnMore": "https://aka.ms/finops/toolkit"
-}
-```
+> ![Version 0.0.1](https://img.shields.io/badge/version-0.0.1-lightgrey) &nbsp; ![Status: Proposed](https://img.shields.io/badge/status-proposed-lightgrey) &nbsp;<sup>→</sup>&nbsp; [![Go to issue](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/34)](https://github.com/microsoft/cloud-hubs/issues/34)
+>
+> 🆕 _Add the following to the storage account:_
+>
+> - containers – Create the following nested containers:
+>   - **config** – Used for configuration settings. Include the following files within the template:
+>     - [settings.json](#settingsjson)
+>   - **ms-cm-exports** – Used for Cost Management exports that have not been transformed.
+>   - **ingestion** – Used for Cost Management exports that have not been transformed.
+>
+> 🆕 _Add the following section:_
+>
+> ### settings.json
+>
+> The **settings.json** file is used to store any configuration settings for the hub instance.
+>
+> ```json
+> {
+>   "$schema": "https://aka.ms/finops/toolkit/settings-schema",
+>   "type": "HubInstance",
+>   "version": "0.0.1",
+>   "learnMore": "https://aka.ms/finops/toolkit",
+>   "exportScopes": <param:exportScopes>
+> }
+> ```
+>
+> ![Version 0.0.1](https://img.shields.io/badge/version-0.0.1-lightgrey) &nbsp; ![Status: Proposed](https://img.shields.io/badge/status-proposed-lightgrey) &nbsp;<sup>→</sup>&nbsp; [![Go to issue](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/59)](https://github.com/microsoft/cloud-hubs/issues/59)
+>
+> 🆕 _Add the following to the ADF instance:_
+>
+> - pipelines – Create the following nested pipelines:
+>   - **ms-cm-exports_Transform** – Converts the CSV export to parquet and moves it to the **ingestion** container (same path), overwriting the existing contents for that month.
+>
+> ![Version 0.0.2](https://img.shields.io/badge/version-0.0.2-lightgrey) &nbsp; ![Status: Proposed](https://img.shields.io/badge/status-proposed-lightgrey) &nbsp;<sup>→</sup>&nbsp; [![Go to issue](https://img.shields.io/github/issues/detail/state/microsoft/cloud-hubs/60)](https://github.com/microsoft/cloud-hubs/issues/60)
+>
+> 🆕 _Add the following ADF pipelines:_
```

Comment:
@MSBrett Let's separate the export setup from the dedupe/parquet work. That way, we can have one smaller check-in that makes the solution functional for v0.0.1 where customers can create their own exports, then follow that up with self-managed exports in v0.0.2. Thoughts?